vNext/UnityGLTFv2.cs

```diff
@@ -175,7 +209,8 @@ public async void LoadGLBs()
     UnityGLTFObject boxObject = new UnityGLTFObject("http://samplemodels/box.glb");

```

Comment:
Is this assuming that deserialization is synchronous? Seems like this would naturally lead people down a bad path.

Reply:
No, this is just passing in a uri that will be resolved during the actual load